vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -73,7 +73,17 @@ def get_quant_method(
         if isinstance(layer, Attention):
             return CompressedTensorsKVCacheMethod(self)
         if isinstance(layer, FusedMoE):
-            return CompressedTensorsMoEMethod(self)
+            # TODO: @dsikka: refactor this to use schemes as other kernels
+            # are supported + check if the layer is being ignored.
```

Comment:
Since we're now expanding the supported schemes, we should have this just return `CompressedTensorsMoEMethod` and have the return the correct MoE scheme within the class since it looks like we're now checking the conditions twice?